casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -379,18 +379,17 @@ object BlockAPI {
 
   def visualizeDag[
       F[_]: Monad: Sync: EngineCell: Log: SafetyOracle: BlockStore,
-      G[_]: Monad: GraphSerializer,
       R
   ](
       depth: Int,
       maxDepthLimit: Int,
       startBlockNumber: Int,
-      visualizer: (Vector[Vector[BlockHash]], String) => F[G[Graphz[G]]],
-      serialize: G[Graphz[G]] => R
+      visualizer: (Vector[Vector[BlockHash]], String) => F[Graphz[F]],
+      serialize: F[Graphz[F]] => R
```

Comment:
I think we can simplify serialize function to `serialize: Graphz[F] => R`.

Reply:
Resolved in [commit](https://github.com/rchain/rchain/pull/3574/commits/3731710108b1717020f2b884491db78dfe7bc0ad)